tests/basic.cpp

```diff
@@ -241,6 +241,9 @@ TEST_CASE("dbg::hex and dbg::oct") {
   uint32_t oct_1 = 01234567;
   CHECK(pretty_print(dbg::oct(oct_1)) == "0o1234567");
 
+  uint8_t bin_1 = 0b1011;
+  CHECK(pretty_print(dbg::bin(bin_1)) == "0b1011");
+
```

Comment:
Can we please add a few more tests:
* negative numbers
* zero padding (see above)
* other integer sizes (e.g. `uint16_t`)

Reply:
For sure!  
I might open an issue for this but will say it here. Seeing as you've just revamped the tests could you make a contributing.md and outline how you want new tests to be added? Outline how you want it laid out, how many you want, etc.